_src/automation/generate_cli_schema_export_pages.py

```diff
@@ -42,7 +46,7 @@ def create_page(name, json_entry):
 		for subcommand_json in json_entry["subcommands"]:
```

Comment:
```python3
qmd = "".join([qmd_add_command(subcommand_jon, is_subcommand=True) for subcommand_jon in json
_entry["subcommands"]])
```

Reply:
This script still needs to be updated to our new system of using jinja templates, so this most likely will be changed. But if not, I'll keep it in mind